include/mxnet/lib_api.h

```diff
@@ -328,16 +358,31 @@ typedef void* (*xpu_malloc_t)(void*, int);
  */
 class OpResource {
  public:
-  OpResource(xpu_malloc_t cm, void* ca) : cpu_malloc(cm), cpu_alloc(ca) {}
+  OpResource(xpu_malloc_t cm, void* ca, xpu_malloc_t gm, void* ga, void* st)
```

Comment:
I know you're just following the existing precedent, but can we change the names `cm` to cpu_mem_alloc so its more readable?